packages/messaging/src/helpers/array-to-base64.test.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { arrayToBase64 } from './array-to-base64';
+import { expect } from 'chai';
+import '../testing/setup';
+
+describe('arrayToBase64', () => {
+  it('returns expected value for p256dh example', () => {
+    // prettier-ignore
+    const buffer = new Uint8Array([
+      4, 181, 98, 240, 48, 62, 75, 119, 193, 227, 154, 69, 250, 216, 53, 110,
```

Comment:
Curious: any particular reason for these numbers? A valid test key? Lottery numbers?

For test code, if the exact values don't matter, it's usually an excuse to use joke data that's obviously unimportant, or at least exercises the edge cases :p

Reply:
No idea, it was before my time. :smile:

https://github.com/firebase/firebase-js-sdk/blob/73a586c92afe3f39a844b2be86086fddb6877bb7/packages/messaging/test/array-buffer-to-base64.test.ts